tutor/src/screens/assignments/details.js

```diff
@@ -44,10 +44,10 @@ const Details = ({ ux }) => {
             <HintText>(Apply a pre-set submission and grading policy template)</HintText>
           </Label>
           <div>
-            <Select name="grading_template">
+            <Select name="grading_template_id">
```

Comment:
@nathanstitt Glad you pointed this out — looking at it again today, I had made an assumption about `grading_template_id` being wired up. Hooking it up gave me what I was looking for.